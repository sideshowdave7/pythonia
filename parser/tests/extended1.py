def a():
 import math
 return math.sqrt(4)

from math import log
from math import log as bar

