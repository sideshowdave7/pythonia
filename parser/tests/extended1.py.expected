(program
 (def (a) (suite (import math) (return (indexed math (dot sqrt) (called 4)))))
 (import math log)
 (import math log bar))

